### PR TITLE
[MM-51453] - Bring back old log list if format is plain text

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
This PR aims to bring back the old UI for the system server logs, when the logs are returned in plain format. Forcing JSON logging was causing issues to our customers who do not have that setting ON. With this change we aim to provide a quick solution to this problem and let people see their logs regardless of formatting preference.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-51453

#### Release Note
```release-note
ADD OLD SERVER LOG UI FOR PLAIN TEXT FORMATTING 
```

### CODE REVIEW

File: server/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "server",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```
Comment (isacikgoz) [inline server/package-lock.json:6]: Is this file necessary?

Comment (nevyangelova) [reply server/package-lock.json:6]: oh my bad.

File: webapp/channels/src/components/admin_console/server_logs/logs.tsx [hunk 2]
```diff
@@ -41,13 +56,34 @@ export default class Logs extends React.PureComponent<Props, State> {
...
+        if (this.props.isPlainLogs) {
+            this.reloadPlain();
+        } else {
+            this.reload();
+        }
     }
 
+    componentDidUpdate(prevProps: Props, prevState: State) {
+        if (this.state.page !== prevState.page) {
+            this.reloadPlain();
```
Comment (harshilsharma63) [inline webapp/channels/src/components/admin_console/server_logs/logs.tsx:75]: Should we add a check for `this.props.isPlainLogs` here to make it explicit? I understand that `this.state.page` changes only when we're in plain logs but adding a check for `this.props.isPlainLogs` makes it explicit and clear for future self.

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
@nevyangelova one minor comment else all looks good. Some CI tasks are failing though.

**Review by AshishDhama — APPROVED** [review]

**Comment by nevyangelova** [comment]
@harshilsharma63 updated. There seems to be a failing test that is not related as i get it on other PRs as well

**Comment by harshilsharma63** [comment]
> @harshilsharma63 updated. There seems to be a failing test that is not related as i get it on other PRs as well

Ah, okay. Looks like a flaky test.

**Review by harshilsharma63 — APPROVED** [review]
LGTM 👍

**Review by isacikgoz — COMMENTED** [review]
This looks good to me functionality-wise but I'm not a front-end expert so take my approval as a grain of salt :)

BTW, the webapp CI seems to be failing, can you check that @nevyangelova
